cmd/faucet/faucet.go

```diff
@@ -235,22 +240,53 @@ func newFaucet(genesis *core.Genesis, port int, enodes []*discv5.Node, network u
 	if err != nil {
 		return nil, err
 	}
-	// Assemble the Ethereum light client protocol
+	// Using Plasma-evm full node as client
 	if err := stack.Register(func(ctx *node.ServiceContext) (node.Service, error) {
-		cfg := pls.DefaultConfig
-		cfg.SyncMode = downloader.LightSync
-		cfg.NetworkId = network
-		cfg.Genesis = genesis
-		return les.New(ctx, &cfg)
+		cfg := pls.Config{
+			Genesis:  genesis,
+			NodeMode: 1,
+			SyncMode: downloader.FullSync,
+			TxConfig: *tx.DefaultConfig,
+			Ethash: ethash.Config{
+				CacheDir:       "ethash",
+				CachesInMem:    2,
+				CachesOnDisk:   3,
+				DatasetsInMem:  1,
+				DatasetsOnDisk: 2,
+			},
+			NetworkId:          genesis.Config.ChainID.Uint64(),
+			LightPeers:         100,
+			UltraLightFraction: 75,
+			DatabaseCache:      512,
+			TrieCleanCache:     256,
+			TrieDirtyCache:     256,
+			TrieTimeout:        60 * time.Minute,
+
+			RootChainURL:      rootchain,
+			RootChainContract: common.BytesToAddress(genesis.ExtraData),
+
+			OperatorMinEther: big.NewInt(0.5 * params.Ether),
+
+			StaminaConfig: core.DefaultStaminaConfig,
+
+			TxPool: core.DefaultTxPoolConfig,
+			GPO: gasprice.Config{
+				Blocks:     20,
+				Percentile: 60,
+			},
+		}
+
+		return pls.New(ctx, &cfg)
```

Comment:
instead of copy & past & fix `pls.defaultConfig`, just update members of `pls.DefaultConfig`, as previous codes.